components/src/core/components/List/List.vue

```diff
@@ -131,6 +131,15 @@
           @update:selected="tableSelect"
           rowDecorator="oxd-table-decorator-card"
         />
+        <div
+          v-if="wholeRecordsCount === 0 && !isListLoading"
+          class="empty-msg-container"
+        >
+          <div class="empty-msg">
+            <oxd-icon class="empty-msg-pic" name="oxd-no-data" />
+            <div class="caption">{{ $t('Sorry, No Data Found!') }}</div>
```

Comment:
It was asked to use Localization Mixin rather than the $t directly in the component when we call the localisation in the template because it'll not work in story book. Check the following PR. 

https://github.com/orangehrm/oxd/pull/315

Reply:
Thank you Thilanka.